securing_apps/topics/overview/basic-steps.adoc

```diff
@@ -0,0 +1,24 @@
+=== Basic steps to secure applications and services
+
+These are the basic steps for securing an application or a service in {project_name}.
+
+. Configure a client using one of these options:
+
+* A {project_name} adapter
+
+* A generic OpenID connect or SAML library 
+
+. Register a client using one of these options:
+
+* The {project_name} Admin Console
+
+* The client registration service
+
+* The CLI
+
+[role="_additional-resources"]
+
+.Additional resources
+
+* This guide provides the detailed instructions for these steps. Additional information on using the Admin Console exists in the link:{adminguide_link}[Server Administration Guide]. That guide refers to creating a client while the Securing Apps and Services Guide refers to registering a client.
```

Comment:
I might be wrong, but registering and creating a client refers to the same thing. So, the last sentence does not sound correct.

Perhaps another option here could be:

```
Additional information on how to manage clients using the Admin console exists in the link:{adminguide_link}[Server Administration Guide].
```